front-end/cypress/e2e/pages/loginPage.ts

```diff
@@ -67,7 +68,20 @@ function loginDotGovLogin() {
   cy.wait('@GetCommitteeAccounts');
   cy.get('.committee-list .committee-info').first().click();
   cy.wait('@ActivateCommittee');
-  //cy.visit('/dashboard');
+
+  // Create second create admin after logging in to make pop-up go away
+  cy.visit('/dashboard');
+  cy.wait(500);
```

Comment:
We've been phasing out fixed-time wait calls, so I took a few minutes to make an alternative solution.  I've opened a PR into this PR [here](https://github.com/fecgov/fecfile-web-app/pull/2610)